packages/gatsby-source-contentful/src/gatsby-node.js

```diff
@@ -58,6 +61,19 @@ exports.sourceNodes = async (
   console.log(`assets fetched`, assets.items.length)
   console.timeEnd(`fetch Contentful data`)
 
+  // Create map of not resolvable ids so we can filter them out while creating
+  // links.
+  const notResolvable = new Map()
+  entryList.forEach(ents => {
+    if (ents.errors) {
```

Comment:
The only place I found it documented is here: http://cocoadocs.org/docsets/ContentfulDeliveryAPI/1.9.2/Classes/CDAArray.html

Basically says if a referenced entry is deleted but the reference also isn't also deleted they send this error 🤷‍♂️ 

The client I'm working with has a ton of content types and has been doing a lot of experimenting so this came up.